build.gradle

```diff
@@ -39,12 +39,19 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.0'
+        classpath 'com.android.tools.build:gradle:3.5.1'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
 
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
     }
+
+    configurations.all {
+        resolutionStrategy {
+            // Want to force this proguard release.
```

Comment:
can you state the reasons? I think we eventually want to unroll this if we bump gradle to a version that's compatable